<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window_Two</class>
 <widget class="QWidget" name="Window_Two">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_input_data">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Palatino Linotype</family>
     <pointsize>36</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #060ADE;
color: White;
border-style: outset;
border-width: 2px;
border-color: black;
border-top: 0;
border-right: 0; 
border-bottom: 2px solid;
 border-left: 0</string>
   </property>
   <property name="text">
    <string>                    INPUT DATA:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_total_credit_amount">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>215</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Code Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
	background-color: rgba(0, 0, 0, 0.1);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_credit_amount">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>440</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Miriam Mono CLM</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TOTAL CREDIT AMOUNT:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_term">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Code Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
	background-color: rgba(0, 0, 0, 0.1);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_term">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>310</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Miriam Mono CLM</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TERM (MONTHS):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_interest_rate">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>435</y>
     <width>350</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Miriam Mono CLM</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>INTEREST RATE:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_interest_rate">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Code Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
	background-color: rgba(0, 0, 0, 0.1);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_type">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>575</y>
     <width>110</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Miriam Mono CLM</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TYPE:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="box_annuity">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>635</y>
     <width>180</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Miriam Mono CLM</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>ANNUITY</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="box_differentiated">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>635</y>
     <width>330</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Miriam Mono CLM</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DIFFERENTIATED</string>
   </property>
  </widget>
  <widget class="QLabel" name="output_data">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>0</y>
     <width>800</width>
     <height>110</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Palatino Linotype</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #8906DE;
color: White;
border-style: outset;
border-width: 2px;
border-color: black;
border-top: 0;
border-right: 0; 
border-bottom: 2px solid;
 border-left: 0</string>
   </property>
   <property name="text">
    <string>                    OUTPUT DATA:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_overpayment_on_credit">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>265</y>
     <width>480</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Miriam Mono CLM</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OVERPAYMENT ON CREDIT:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_overpayment_on_credit">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>320</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Code Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
	background-color: rgba(0, 0, 0, 0.1);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_total_payment">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>430</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Code Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
	background-color: rgba(0, 0, 0, 0.1);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_payment">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>375</y>
     <width>300</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Miriam Mono CLM</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TOTAL PAYMENT:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_monthly_payments">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>160</y>
     <width>590</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Miriam Mono CLM</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MONTHLY PAYMENTS (average):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_line">
   <property name="geometry">
    <rect>
     <x>798</x>
     <y>0</y>
     <width>4</width>
     <height>900</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: Black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button_calculate">
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>450</y>
     <width>200</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semilight</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
	background-color: rgba(0, 0, 0, 0.1);</string>
   </property>
   <property name="text">
    <string>CALCULATE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_monthly_payments">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>215</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Code Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: black;
background-color: rgba(0, 0, 0, 0.1);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_bigger_then">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>320</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(* more then 10 and less then 1 000 000 000)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_bigger_then2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>395</y>
     <width>220</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(* more then 0 and less then 600)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_bigger_then3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>535</y>
     <width>220</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(* more then 0 and less then 300)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_payments">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>485</y>
     <width>191</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Miriam Mono CLM</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PAYMENTS:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>-10</y>
     <width>1600</width>
     <height>900</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/window_two_background.jpg</pixmap>
   </property>
  </widget>
  <widget class="QListWidget" name="list_payments">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>540</y>
     <width>600</width>
     <height>275</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Serif Lao</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
border-width: 2px;
border-radius: 10px;
border-color: black;
background-color: rgba(0, 0, 0, 0.1);</string>
   </property>
  </widget>
  <zorder>label_background</zorder>
  <zorder>label_input_data</zorder>
  <zorder>line_total_credit_amount</zorder>
  <zorder>label_total_credit_amount</zorder>
  <zorder>line_term</zorder>
  <zorder>label_term</zorder>
  <zorder>label_interest_rate</zorder>
  <zorder>line_interest_rate</zorder>
  <zorder>label_type</zorder>
  <zorder>box_annuity</zorder>
  <zorder>box_differentiated</zorder>
  <zorder>output_data</zorder>
  <zorder>label_overpayment_on_credit</zorder>
  <zorder>line_overpayment_on_credit</zorder>
  <zorder>line_total_payment</zorder>
  <zorder>label_total_payment</zorder>
  <zorder>label_monthly_payments</zorder>
  <zorder>label_line</zorder>
  <zorder>button_calculate</zorder>
  <zorder>line_monthly_payments</zorder>
  <zorder>label_bigger_then</zorder>
  <zorder>label_bigger_then2</zorder>
  <zorder>label_bigger_then3</zorder>
  <zorder>label_payments</zorder>
  <zorder>list_payments</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
